script/travis/run_build.sh

```diff
@@ -55,6 +55,10 @@ echo "KMP_ALL_THREADS=${KMP_ALL_THREADS}"
 echo "KMP_TEAMS_THREAD_LIMIT=${KMP_TEAMS_THREAD_LIMIT}"
 echo "OMP_THREAD_LIMIT=${OMP_THREAD_LIMIT}"
 echo "OMP_NUM_THREADS=${OMP_NUM_THREADS}"
+echo "ALPAKA_CI_CATCH2_ROOT_DIR=${ALPAKA_CI_CATCH2_ROOT_DIR}"
+ls ${ALPAKA_CI_CATCH2_ROOT_DIR}
+ls -R ${ALPAKA_CI_CATCH2_ROOT_DIR}
```

Comment:
Can we remove those traces?

Reply:
Yes, will clean up as soon as the CI passes.